### ISSUE TITLE
Installation issues #2 (CUDA, Detectron2, etc)

### ISSUE THREAD

**Issue by jeonhuhuhu** [notmahi/clip-fields#8]
Here is an error that occurred before updating GitHub that was not present before. 

1. When installing with the code provided in the installation manual, a Detectron2 installation error occurs. 
![Screenshot from 2024-03-04 09-47-13](https://github.com/notmahi/clip-fields/assets/148358798/e457b03a-38e9-4fbc-8b7c-dcf6b4443e4c)

- conda install pytorch torchvision torchaudio pytorch-cuda=11.8 -c pytorch -c nvidia
- `conda install -y pytorch torchvision torchaudio cudatoolkit=11.8 -c pytorch-lts -c nvidia`

To avoid the Detectron2 error, 
we need to create a Conda environment using the second code.

2. When completing the installation and proceeding with training, a "ModuleNotFoundError" error occurs. (ModuleNotFoundError: No module named 'additional_utils')
![Screenshot from 2024-03-04 10-47-44](https://github.com/notmahi/clip-fields/assets/148358798/56c8345c-fa30-4da7-b6c3-8a7e7ec04446)

I find OK-Robot research fascinating, 
I've been experimenting with various installation methods and troubleshooting. 

When installing with the updated code from two days ago, additional errors occurred.

However, when conducting experiments with the GitHub repository downloaded two days ago, 
there seem to be no issues at all.

**Comment by notmahi** [comment]
Did you _recursively_ clone all the submodules as well? It seems like the `lseg` modules are not present which are a submodule to the main module. The right command should be: `git clone --recursive https://github.com/notmahi/clip-fields`

**Comment by jeonhuhuhu** [comment]
> Did you _recursively_ clone all the submodules as well? It seems like the `lseg` modules are not present which are a submodule to the main module. The right command should be: `git clone --recursive https://github.com/notmahi/clip-fields`

"Of course, we did download the GitHub in that manner, 
but when running the "python setup.py install" code for "gridencoder" installation, the same error as before occurs. 
command: `git clone --recursive https://github.com/notmahi/clip-fields`

However, as suggested in the previous issue, resolving it in a similar manner should suffice. 
So please note that even with the GitHub you have newly updated, the same error still occurs."

--------------------------------------------------------------

I have successfully completed the training using the previous code and generated command: `detic_labeled_dataset.pt`.

Therefore, to conduct our test, 
should we simply insert our trained ".pt" file into the `4-test model.ipynb` section under the Demo area? 
`training_data = torch.load("../detic_labeled_dataset.pt")`

In other words, is "4 - test model.ipynb" the final outcome we can expect from this research?

If we intend to integrate with Hello-Robots, should we refer to OK-Robot?"

**Comment by notmahi** [comment]
Good to hear that you have fixed the issue, although I am not sure what exactly is the "similar manner" you have referred to.

> In other words, is "4 - test model.ipynb" the final outcome we can expect from this research? If we intend to integrate with Hello-Robots, should we refer to OK-Robot?

Yes, and you can integrate the results of `4 - test model.ipynb` with a robot controller if you want. However, if you want a ready-made robot integration, you should be looking at the OK-Robot repo: https://github.com/ok-robot/ok-robot

**Comment by jeonhuhuhu** [comment]
> Good to hear that you have fixed the issue, although I am not sure what exactly is the "similar manner" you have referred to.
> 
> > In other words, is "4 - test model.ipynb" the final outcome we can expect from this research? If we intend to integrate with Hello-Robots, should we refer to OK-Robot?
> 
> Yes, and you can integrate the results of `4 - test model.ipynb` with a robot controller if you want. However, if you want a ready-made robot integration, you should be looking at the OK-Robot repo: https://github.com/ok-robot/ok-robot

Thank you for your assistance. 

A detailed solution can be found in Section https://github.com/notmahi/clip-fields/issues/7 .
